specification/columns/servicesubcategory.md

```diff
@@ -0,0 +1,102 @@
+|                           | Data Processing                   | Integration and transformation tasks to prepare data for analysis.                                                                                                                                     |
+|                           | Search                            | Discovery of information by indexing and retrieving data from various sources.                                                                                                                         |
+|                           | Streaming Analytics               | Real-time data stream processes to detect patterns, trends, and anomalies as they occur.                                                                                                               |
+|                           | Other (Analytics)                 | Analytics services that do not fall into one of the defined subcategories.                                                                                                                             |
+| Business Applications     | Other (Business Applications)     | All Business Applications services.                                                                                                                                                                    |
+| Compute                   | Containers                        | Management and orchestration of containerized compute platforms.                                                                                                                                       |
+|                           | End User Computing                | Virtualized desktop infrastructure and device / endpoint management.                                                                                                                                   |
+|                           | Quantum                           | Resources and simulators that leverage the principles of quantum mechanics.                                                                                                                            |
+|                           | Serverless Compute                | Enablement of compute capabilities without provisioning or managing servers.                                                                                                                           |
+|                           | Virtual Machines                  | Computing environments created by abstracting resources from physical hardware.                                                                                                                        |
+|                           | Other (Compute)                   | Compute services that do not fall into one of the defined subcategories.                                                                                                                               |
```

Comment:
This was debated for a long time in many forums (TF1, maintainer, member). 

I think the final decision was it would go in Other depending on how broadly the discount instrument applied. If it was specific to some category, e.g. Compute / Virtual Machines, then it would go there. If it was a general compute or database RI/SP, would go into Other ($Category). 

@ijurica @salpaysenturus please correct if I'm off base here.

Reply:
That is correct.  The first draft included a `Crossfunctional` category and category-specific `Commitment` subcategories to handle for cross-category items.  This was very heavily discussed, particularly on the Jul 23 TF1 and Aug 8 Member calls, and the outcome for commitments was:

1) Add to the relevant Subcategory where possible (e.g. RDS RI goes to `Compute > Virtual Machines`); 
2) Add to a Category-specific Uncategorized Subcategory where possible (e.g. Compute SP goes to `Compute > Other (Compute)`); 
3) Do not categorize or subcatgorize where it crosses Categories (e.g. OCI Credits goes to `Other > Other (Other)`).

Reply:
Aligned on the arrangement described in my previous comment, which does not attempt to categorize or subcategorize Commitment Discounts.  Resolving.